gallery/src/components/NxNexusPageHeader/NxNexusPageHeaderExample.tsx

```diff
@@ -26,7 +26,8 @@ const NxNexusPageHeaderExample = () => {
   return (
     <NxNexusPageHeader productInfo={{ name: 'Test Product', version: 'Fancy code name 1.0.0' }}
                        homeLink="#"
-                       links={links}>
+                       links={links}
+                       imgURL="./resources/logos/logo_nexus_generic.svg">
```

Comment:
I think webpack needs to be used for this to ensure the path is right.

Reply:
> I think webpack needs to be used for this to ensure the path is right.

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/669e2463187549821dd756fda3c7279e294a1c95

Reply:
> 669e246

Please move it to the top level of the file

Reply:
> > [669e246](https://github.com/sonatype/sonatype-react-shared-components/commit/669e2463187549821dd756fda3c7279e294a1c95)
> 
> Please move it to the top level of the file

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/156df1fb89516f53c9f7b74cc9d6b15440c5c072